utils/node_local_test.py

```diff
@@ -2335,8 +2335,12 @@ def test_cont_rw(self):
         print(ps)
 
         with open(join(dfuse.dir, 'rw_dir', 'new_file'), 'r') as fd:
-            data = fd.read()
+            data = fd.read(14)
             print(data)
+            if data != 'read-only-data':
+                fd.close()
```

Comment:
Tests expect close() called automatically in __exit_.

Reply:
The "with" construct on L2337 will mean that fd is automatically closed as it goes out of scope.